src/go/rpk/pkg/tuners/executors/commands/write_sized_file.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// +build linux
+
+package commands
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
+)
+
+type writeSizedFileCommand struct {
+	path      string
+	sizeBytes int64
+}
+
+func NewWriteSizedFileCmd(path string, sizeBytes int64) Command {
+	return &writeSizedFileCommand{path, sizeBytes}
+}
+
+func (c *writeSizedFileCommand) Execute() error {
+	log.Debugf("Creating  '%s' (%d B)", c.path, c.sizeBytes)
```

Comment:
two spaces after `Creating`

Reply:
Fixed. I also added a comment to the constructor describing the command's behavior.